gameofgame.py

```diff
@@ -58,44 +58,60 @@ def check_guess(guess, country):
     # Both Players take turns to play
     # player1 goes first
     print("Player 1's turn")
-    player1_guess = input(f"What is the capital of {country}? ")
-    player1_guess = getpass.getpass(prompt=f"What is the capital of {country}? ")
+    # player1_guess = input(f"What is the capital of {country}? ")
+    player1_guess = getpass.getpass(
+        prompt=f"What is the capital of {country}? ")
     # player 2 goes next
     print(player_turn_str.format(player_num=2))
-    player2_guess = input(f"What is the capital of {country}? ")
+    # player2_guess = input(f"What is the capital of {country}? ")
     # player 2 goes next
-    print("Player 2's turn")
-    player2_guess = getpass.getpass(prompt=f"What is the capital of {country}? ")
+    #print("Player 2's turn")
+    player2_guess = getpass.getpass(
+        prompt=f"What is the capital of {country}? "
+        )
 
     # Reveal the correct answer
     capital = countries_capitals[country]
     print(f"The correct answer is {capital}.")
 
+    # Check the guesses and update the scores
+    # print(f"The correct answer is {capital}.")
+
     # Check the guesses and update the scores
     if check_guess(player1_guess, country) and check_guess(player2_guess, country):
         print("Both players guessed correctly!")
         player1_score += 1
-        player2_score += 1
+        player2_score += 1     
     elif check_guess(player1_guess, country):
         print("Player 1 guessed correctly!")
         print("Well done, you know your Geography well!")
-        player1_score += 1 
+        player1_score += 1
+        
     elif check_guess(player2_guess, country):
         print("Player 2 guessed correctly!")
         print("Well done, you know your Geography well!")
 
         player2_score += 1
     else:
         print("Neither player guessed correctly.")
-        print("You both should learn about capital cities in the world!!!")     
+        print("You both should learn about capital cities in the world!!!")
+        
     # Print the current scores
     print(
         f"Current scores: Player 1: {player1_score}, Player 2: {player2_score}"
         )
+    
     # Ask if the players want to continue playing
     play_again = input("Do you want to play again? (y/n) ")
     if play_again.lower() != "y":
-        break
+    
+
+    # Determine the winner and print the congratulatory message
+        if player1_score > player2_score:
+            print(congrats_message.format(winner=1))
+        elif player2_score > player1_score:
+            print(congrats_message.format(winner=2)) 
     else:
-        print("The game ended in a tie. Well played both players!")  
-print("Thanks for playing!")
\ No newline at end of file
+        print("The game ended in a tie. Well played both players!")
+    print("Thanks for playing!")
```

Comment:
You always print this one, regardless of whether the players decide to play again or not.